README.md

```diff
@@ -17,7 +17,7 @@ is a fork of [bcoin][bcoin] and has the same RPC API.
 - Mining Backend (getblocktemplate support)
 - General Purpose Bitcoin Library
 
-Try it in the browser: http://bcoin.io/browser.html
+Try it in the browser: https://bcoin.io/browser.html
```

Comment:
This link is incorrect, It must be https://bcoin.io/browser/

Reply:
That's right. Buck just updated the link yesterday in bcoin. The actual endpoint is bcoin.io/browser/index.html

That page is all bcoin though! But I can add all the same options for bcash as well.